docs/en/introduction/Dora-Architecture.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
i am not in favor of readding this page. the readded content is more in the form of release notes or blog post, not documentation